Document stderr redirection for shell_output tests (#19120)
Homebrew/brew: docs/Formula-Cookbook.md

end
```

> * If your program writes to sterr, you can [redirect sterr to stdout for assertions with `shell_output`](https://github.com/Homebrew/brew/issues/5149). For example:

Reviewer: ```suggestion
* If the binary only writes to `stderr`, you can redirect `stderr` to `stdout` for assertions with `shell_output`. For example:
```
